benches/basic.rs

```diff
@@ -126,3 +133,75 @@ bench_over_inputs!(eq, |b, input| {
     let second = black_box(first.clone());
     b.iter(|| first == second);
 });
+
+macro_rules! contended_from_str {
```

Comment:
Several of these are printing warnings that the catch-all configuration I used isn't the best idea:

```
Warning: Unable to complete 100 samples in 5.0s. You may wish to increase target time to 7.6s, enable flat sampling, or reduce sample count to 50.
```